skulpt.py

```diff
@@ -1312,13 +1316,13 @@ def main():
         f.write(getInternalCodeAsJson() + ";")
 
     if cmd == "test":
-        test()
+        exit(bool(test()))
```

Comment:
Do these get the success/failure thing the right way round? I thought `exit()` took an integer, and 0 was success and 1 was failure...

Reply:
`test()` returns non zero for something bad

`bool(0) => False => 0` which is OK
`bool(>0) => True => 1` which is not OK

Reply:
Approved, then!

Reply:
The problem I was having is that jscs returns `512` on error and `exit(512)` results in a `0` return code :trollface:

Reply:
[Head->Desk]